tests/qunit.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="utf-8">
+	<title>QUnit Example</title>
+
+	<link rel="stylesheet" href="qunit/qunit-1.19.0.css">
+
+	<script src="../js/jquery-2.1.4.min.js"></script>
+	<script src="qunit/qunit-1.19.0.js"></script>
+
+	<script src="../fundraising/banner.js"></script>
+	<script src="../fundraising/banner.config.js"></script>
+	<script type="text/javascript">
+		Banner.config.tracking.libUrl = 'qunit/piwik-dummy.js';
```

Comment:
I agree to your opinion. If we put it here it's applying this configuration to all tests (which may not be what we want).